index.js

```diff
@@ -362,8 +357,12 @@ export var memo = (tag, memo) => ({ tag, memo })
 export var text = (value, node) =>
   createVNode(value, EMPTY_OBJ, EMPTY_ARR, TEXT_NODE, node)
 
-export var h = (tag, props, children = EMPTY_ARR) =>
-  createVNode(tag, props, isArray(children) ? children : [children])
+export var h = (tag, { class: c, ...p }, children = EMPTY_ARR) =>
```

Comment:
Question: Does `{ ...p, class: c }` work as well ?

If so it might be worth considering using it to have some symmetry with line 363.

Reply:
Can we use a default value for `c` to avoid the `!obj ||` inside `createClass` and would it be worth it?

Reply:
Yes I fixed that now In my own way of checking that means this diff is just +2B (minified and gzipped), So I'd say it looks worth it ;) 

Caveat is I still don't know how this affects performance. I feel pretty confident this does not affect perf at all but I haven't verified. I could run the krause bench test but since we memoize for that test any perf impact of this wouldn't show anyway. 

I'd be happy to do a comparison without memoization, but I don't have the bandwidth for it the next week. For myself I'm fine awaiting those results before merging.

Reply:
Thanks, @zaceno. I say we merge this and _see_. If perf is affected, we can go back to no default arguments.